index.html

```diff
@@ -1481,16 +1401,16 @@ <h2>Some more "projects"</h2>
       </div>
     </div>
 
-    <style>
-      .c-site-footer { 
-        background-color: var(--brand-red); color:white; padding: 40px 20px
-      }
-      .c-site-footer a { color: var(--brand-white); text-decoration: none; border-bottom: 2px solid var(--brand-black); padding:2px}
-    </style>
-    <footer class="c-site-footer">
+    <footer>
       <h3>CYF Blocks is a project of <a href="https://codeyourfuture.io">Code Your Future</a></h3>
       <p>Recap your HTML/CSS knowledge with this <a href="https://scrimba.com/learn/htmlcss">free crash course</a> on Scrimba.</p>
     </footer>
+     <!-- <dialog aria-live="assertive" class="toast" id="toast" aria-labelledby="toast-title" closed>
```

Comment:
toast?

Reply:
If I get 20 minutes it will manifest there.

Reply:
toast